codec/collections.go

```diff
@@ -79,7 +84,51 @@ func (c collValue[T, PT]) Stringify(value T) string {
 }
 
 func (c collValue[T, PT]) ValueType() string {
-	return "gogoproto/" + proto.MessageName(PT(new(T)))
+	return "github.com/gogo/protobuf/" + c.messageName
```

Comment:
Just curious, shouldn’t it be our fork?

Reply:
I don't think it's that important tbh, in theory even prepending this info is not needed, it's like the any type URL where only the string after the last `/`  matters.

Reply:
Why not setting our fork name then instead?